addons/base_onboarding/models/base_onboarding.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from .base_onboarding_state import ONBOARDING_STATES
+from odoo.tools.json import scriptsafe as json_safe
+
+
+class Onboarding(models.Model):
+    _name = 'base.onboarding'
```

Comment:
After long discussions about model, universe, life, ... with @awa-odoo we propose to remove the ``base`` prefix (which does not mean much), and instead use ``onboarding``, aka: ``onboarding.definition`` (main model), ``onboarding.definition.step``

Reply:
For actual usage, I suggest: ``onboarding.progress`` and ``onboarding.progress.step``, as a global proposal to replace and/or improve the state mixin that is not crystal clear in my mind :sweat_smile: